apps/files/src/components/FilesAppBar.vue

```diff
@@ -1,54 +1,52 @@
 <template>
-  <div>
-  <file-drop :url='url' :headers="headers" @success="onFileSuccess" @error="onFileError" @progress="onFileProgress" />
-  <oc-topbar variation="secondary">
-    <template slot="left">
-      <oc-topbar-logo icon="home" @click="navigateTo('files-list', '')"></oc-topbar-logo>
```

Comment:
We are now missing some breadcrumb for root folder. Either keep the button there or we need to add home into the breadcrumbs.

Reply:
Alright, there is this `firstIsHome` feature that goes like this:
https://github.com/owncloud/owncloud-design-system/blob/master/src/elements/OcBreadcrumb.vue#L39

Need to take care of how that should work ;-)

Reply:
Can't we just keep the icon there with the click method? This way if we would in some case needed to disable breadcrumbs, we would still have the icon independent. Doesn't need to be `oc-topbar-logo` but just `oc-icon` should work fine, right? Or even just `oc-button` with an icon when icon wouldn't be enough.

Reply:
Here we go: https://github.com/owncloud/owncloud-design-system/pull/302